### DISCUSSION TITLE
When loading a saved flight, is the MCDU expected to be fully reset?

### DISCUSSION THREAD

**Discussion by rageshkrishna** [flybywiresim/aircraft#7213] [Help] [answered]
Hi folks,

I'm new here, so first of all, what a fantastic product and great community here!

I've been following the beginner guide for the EGFF/EGCC route. I wanted to save the flight so I could keep practicing the manual leg and approach into EGCC. 

I used the MSFS "save" feature from the pause screen. Unfortunately, when I load the flight, the MCDU is completely reset. It's quite literally at the blank-slate state that I get on the ground when I start my planning. The rest of the plane is exactly as I saved it (right location, altitude, speed, etc.).

This puts a dent on my plans to keep practicing the last leg of the flight, so I was wondering if this is a known issue, or limitation, or if there's some way I can get it to work.

Thanks,
-- Ragesh.

**Comment by tracernz** [comment, marked as answer]
Unfortunately this is not possible at the moment.
